pennylane_lightning/core/src/gates/Constant.hpp

```diff
@@ -94,6 +94,47 @@ namespace Pennylane::Gates::Constant {
         GateOperation::DoubleExcitationPlus, "DoubleExcitationPlus"},
     std::pair<GateOperation, std::string_view>{GateOperation::MultiRZ,
                                                "MultiRZ"}};
+
+[[maybe_unused]] constexpr std::array controlled_gate_names = {
```

Comment:
So, just for my knowledge, any gate in this list can be arbitrarily controlled with any count of control wires?

Reply:
Yes.